tests/unittest/_torch/auto_deploy/unit/multigpu/transformations/library/test_allreduce_residual_rmsnorm_fusion.py

```diff
@@ -35,8 +35,8 @@ def __init__(self, hidden_size, dtype):
         self.norm = RMSNorm(hidden_size, 1e-5, dtype)
 
     def forward(self, x, residual):
-        x = torch.ops.auto_deploy.torch_dist_all_reduce(x)
-        y = x + residual
+        x = torch.ops.auto_deploy.torch_dist_all_reduce.default(x)
+        y = residual + x
```

Comment:
any reason why `y = x + residual` doesn't work anymore?

Reply:
seems useful to have :) Easy to get tripped up otherwise and trivial to add another pattern right?